src/main/java/at/ac/tuwien/kr/alpha/solver/MixedRestartStrategy.java

```diff
@@ -0,0 +1,81 @@
+package at.ac.tuwien.kr.alpha.solver;
+
+import static at.ac.tuwien.kr.alpha.Util.oops;
+import static at.ac.tuwien.kr.alpha.solver.NoGoodStore.LBD_NO_VALUE;
+
+/**
+ * A restart strategy that mixes dynamic restarts and Luby sequence-based restarts.
+ *
```

Comment:
Please provide (a) literature reference(s).

Reply:
The literature reference would be:
`Nordström, J., 2019, Personal communication.` :-)

But there is actually something still missing, namely the disabling of restarts if the solver is "close" to an full assignment. Have to look that up and add it.